TurkishFilter/sections/general_extensions.txt

```diff
@@ -239,6 +239,8 @@ yabancidizi.org#$##rek-container { display: none!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/43170
 trabzonhaber24.com#$#.modal-backdrop { display: none !important; }
 trabzonhaber24.com#$#body { overflow: visible !important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/43962
+cazkolik.com#$##solust { position: static!important; }
```

Comment:
why we need this rule?

Reply:
When the other items are hidden, without the rule the text of the radio are not in their boxes.